test/Model/ArticlePoATest.php

```diff
@@ -3,34 +3,233 @@
 namespace test\eLife\ApiSdk\Model;
 
 use DateTimeImmutable;
-use eLife\ApiSdk\Collection\Sequence;
+use eLife\ApiSdk\Collection\ArraySequence;
 use eLife\ApiSdk\Model\ArticlePoA;
-use eLife\ApiSdk\Model\ArticleVersion;
-use GuzzleHttp\Promise\PromiseInterface;
+use eLife\ApiSdk\Model\ArticleSection;
+use eLife\ApiSdk\Model\Block\Paragraph;
+use eLife\ApiSdk\Model\Copyright;
+use eLife\ApiSdk\Model\Model;
+use eLife\ApiSdk\Model\PersonAuthor;
+use eLife\ApiSdk\Model\PersonDetails;
+use eLife\ApiSdk\Model\Subject;
+use PHPUnit_Framework_TestCase;
+use test\eLife\ApiSdk\Builder;
 
-final class ArticlePoATest extends ArticleTest
+final class ArticlePoATest extends PHPUnit_Framework_TestCase
 {
-    protected function createArticleVersion(
-        string $id,
-        int $version,
-        string $type,
-        string $doi,
-        string $authorLine,
-        string $titlePrefix = null,
-        string $title,
-        DateTimeImmutable $published,
-        DateTimeImmutable $statusDate,
-        int $volume,
-        string $elocationId,
-        string $pdf = null,
-        Sequence $subjects,
-        array $researchOrganisms,
-        PromiseInterface $abstract,
-        PromiseInterface $issue,
-        PromiseInterface $copyright,
-        Sequence $authors
-    ) : ArticleVersion {
-        return new ArticlePoA($id, $version, $type, $doi, $authorLine, $titlePrefix, $title, $published, $statusDate,
-            $volume, $elocationId, $pdf, $subjects, $researchOrganisms, $abstract, $issue, $copyright, $authors);
+    /** @var Builder */
+    protected $builder;
+
+    public function setUp()
+    {
+        $this->builder = Builder::for(ArticlePoA::class);
```

Comment:
So this test is using the default values as an input, but it's asserting on some precise expected values. You may want to force the interesting value for each test with a `with...()` or `withPromiseOf...()` before each `__invoke()`.